### PR TITLE
[DEV-5296] Adds total_face_value_loan_amount to /api/v2/recipient/state/…

### PR DESCRIPTION
**Pull request by collinwr**
…{fips}/{?year} endpoint

**Description:**
This adds an aggregation of `face_value_loan_guarantee` to the following endpoint:
[/api/v2/recipient/state/{fips}/{?year}](https://github.com/fedspendingtransparency/usaspending-api/blob/dev/usaspending_api/api_contracts/contracts/v2/recipient/state/fips.md)

**Requirements for PR merge:**

1. [ ] Unit & integration tests updated
2. [x] API documentation updated
3. [ ] Necessary PR reviewers:
    - [ ] Backend
    - [ ] Frontend
4. N/A - Matview impact assessment completed
5. [ ] Frontend impact assessment completed
6. [ ] Data validation completed
7. N/A - Appropriate Operations ticket(s) created
8. [ ] Jira Ticket [DEV-5296](https://federal-spending-transparency.atlassian.net/browse/DEV-5296):
    - [x] Link to this Pull-Request
    - [ ] Performance evaluation of affected (API | Script | Download)
    - [ ] Before / After data comparison

**Area for explaining above N/A when needed:**
```
```

### CODE REVIEW

File: usaspending_api/recipient/v2/views/states.py [hunk 1]
```diff
@@ -53,8 +53,12 @@ def obtain_state_totals(fips, year=None, award_type_codes=None, subawards=False)
...
             matview_search_filter(filters, SummaryStateView)
             .values("pop_state_code")
-            .annotate(total=Sum("generated_pragmatic_obligation"), distinct_awards=StringAgg("distinct_awards", ","))
-            .values("distinct_awards", "pop_state_code", "total")
+            .annotate(
+                total=Sum("generated_pragmatic_obligation"),
+                distinct_awards=StringAgg("distinct_awards", ","),
+                face_value_of_loan=Sum("face_value_loan_guarantee"),
+            )
+            .values("distinct_awards", "pop_state_code", "total", "face_value_loan_guarantee")
```
Comment (tony-sappe) [inline usaspending_api/recipient/v2/views/states.py:61]: Doesn't this value need to list the annotated field `face_value_of_loan` and not the db column?

File: usaspending_api/api_contracts/contracts/v2/recipient/state/fips.md [hunk 2]
```diff
@@ -36,4 +36,5 @@ This endpoint returns a high-level overview of a specific state or territory, gi
     The year the median household income is based on.
 + `total_prime_amount`: 300200000000 (required, number)
 + `total_prime_awards`: 327721 (required, number)
++ `face_value_loan_guarantee`: 88888.0 (required, number)
```
Comment (tony-sappe) [inline usaspending_api/api_contracts/contracts/v2/recipient/state/fips.md:39]: I see examples of  `face_value_loan_guarantee` in the API docs, but this feature request mirrors the new functionality in Recipient Profile (`/api/v2/recipient/duns/{recipient_id}/{?year}`) which use "total_face_value_loan_amount" and "total_face_value_loan_transactions" which is what I recommend for this endpoint

File: usaspending_api/api_contracts/contracts/v2/recipient/state/fips.md [hunk 3]
```diff
@@ -36,4 +36,5 @@ This endpoint returns a high-level overview of a specific state or territory, gi
...
 + `award_amount_per_capita`: 916023.08 (required, number)
```
Comment (tony-sappe) [inline usaspending_api/api_contracts/contracts/v2/recipient/state/fips.md:40]: I think the mock will require count of award loans in the API response as well.

Comment (collinwr) [reply usaspending_api/api_contracts/contracts/v2/recipient/state/fips.md:41]: I added the count

File: usaspending_api/recipient/v2/views/states.py (hunk 1 above)
Comment (collinwr) [reply usaspending_api/recipient/v2/views/states.py:61]: I made the fields match

File: usaspending_api/api_contracts/contracts/v2/recipient/state/fips.md (hunk 2 above)
Comment (collinwr) [reply usaspending_api/api_contracts/contracts/v2/recipient/state/fips.md:39]: Clarified this with @ross-williford. We're going with `total_face_value_loan_amount`

File: usaspending_api/recipient/v2/views/states.py [hunk 4]
```diff
@@ -53,22 +53,30 @@ def obtain_state_totals(fips, year=None, award_type_codes=None, subawards=False)
...
             matview_search_filter(filters, SummaryStateView)
             .values("pop_state_code")
-            .annotate(total=Sum("generated_pragmatic_obligation"), distinct_awards=StringAgg("distinct_awards", ","))
-            .values("distinct_awards", "pop_state_code", "total")
+            .annotate(
+                total=Sum("generated_pragmatic_obligation"),
+                distinct_awards=StringAgg("distinct_awards", ","),
+                face_value_of_loan_guarantee=Sum("face_value_loan_guarantee"),
+            )
+            .values("distinct_awards", "pop_state_code", "total", "face_value_loan_guarantee")
```
Comment (emily-brents) [inline usaspending_api/recipient/v2/views/states.py:61]: The summed value is called `face_value_of_loan_guarantee`, but the value you're calling in `values` and for the results is still `face_value_loan_guarantee`.

File: usaspending_api/recipient/v2/views/states.py [hunk 5]
```diff
@@ -53,22 +53,36 @@ def obtain_state_totals(fips, year=None, award_type_codes=None, subawards=False)
...
-            .values("distinct_awards", "pop_state_code", "total")
+            .annotate(
+                total=Sum("generated_pragmatic_obligation"),
+                distinct_awards=StringAgg("distinct_awards", ","),
+                total_face_value_loan_amount=Sum("face_value_loan_guarantee"),
+            )
+            .values("distinct_awards", "pop_state_code", "total", "total_face_value_loan_amount")
         )
 
+        total_face_value_loan_prime_awards = queryset.filter(type__in=["07", "08"]).count()
```
Comment (emily-brents) [inline usaspending_api/recipient/v2/views/states.py:64]: When testing locally, this value was always `1`, likely due to having this filter come after the annotation/ summing of the total loan amount.

**Review by kwhickey — APPROVED** [review]
Looks good with new named response fields.

**Review by dpb-bah — APPROVED** [review]

**Pull request merged**